notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p><code>or flake rate by tests git</code>,  did you mean <code>or flake rate by tests.</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yeah, I will fix that</p>